examples/run/alpaka/CMakeLists.txt

```diff
@@ -38,7 +35,7 @@ add_library( traccc_examples_alpaka STATIC
    "full_chain_algorithm.hpp"
    "full_chain_algorithm.cpp" )
 target_link_libraries( traccc_examples_alpaka
-   PUBLIC alpaka::alpaka vecmem::core detray::core detray::detectors
+   PUBLIC vecmem::core detray::core detray::detectors
```

Comment:
`alpaka` calls now removed from the `examples` CMakeLists.

Reply:
:+1: